spec/features/admin_create_class_category_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+feature 'Admin create class category' do
+  scenario 'sucessfully' do
+    user = User.create!(email: 'renata@smartflix.com.br', password: '123456', role: :admin)
```

Comment:
Como o usuário não parece algo importante depois você pode trocar isso pra uma Trait no FactoryBot aí daria pra fazer só um:

```suggestion
    user = create(:user, :admin)
```

Ficaria bem mais simples pra não ter que escrever tudo hehe

Reply:
Isso vale pra vários testes